flask_lesson_intro/main.py

```diff
@@ -9,5 +10,19 @@ def get_home_page():
     return render_template("home.html", data=get_data())
 
 
+@app.route('/author')
+def get_author_page():
+    return render_template("author.html")
+
+
+@app.route('/<item>')
+def get_items_page(item):
+    data = get_data()
+    for elem in data:
+        if item == elem['title']:
+            return render_template("items.html", title=elem['title'], text=elem['text'],
```

Comment:
what if yoou do not find satisfied element? Take a look at `for..else`

Reply:
added exception handling, thanks